Makefile.am

```diff
@@ -4,7 +4,7 @@ if ENABLE_JPEG
 jpeg_DIRS = jpegutils
 endif
 
-SUBDIRS = $(jpeg_DIRS) cut-n-paste src man plugins po help data doc
+SUBDIRS = po $(jpeg_DIRS) cut-n-paste src man plugins help data doc
```

Comment:
Why sorting `po` to first position?

Reply:
I think the .po file needs to have been generated before creating the desktop file from the desktop.in template. Is it wrong?

Reply:
Ok, good.